about/policy/index.md

```diff
@@ -31,11 +31,6 @@ UGRC’s guidelines for creating, editing, maintaining, and removing data in the
 
 UGRC’s guidelines for using the Open SGID database.
 
-#### [UGRC Data License and Disclaimer]({% link about/policy/license-disclaimer/index.md %})
-{: .text-left}
-
-UGRC’s data license and general data disclaimer.
-
```

Comment:
**question**: did this get moved or is it a bad rebase?

Reply:
The order got moved around in another branch and pushed to master. I rebased at some point after that and probably mixed it up there. I've checked the other pages/changes and they look good, so I think that's the only snafu (famous last words...)

Reply:
And yes, I want to run this by the larger group as well given the amount of changes and the very specific policy wording. I think everyone's already seen the first version, but it's been long enough that a fresh look could prove useful.